lib/livebook/live_markdown/export.ex

```diff
@@ -391,16 +402,27 @@ defmodule Livebook.LiveMarkdown.Export do
     end
   end
 
-  defp ensure_order(%{} = map) when not is_struct(map) do
-    map
+  # Wraps JSON.protocol_encode/2 to encode maps as sorted objects
+  defp encode_sorting(term, encoder) when is_non_struct_map(term) do
```

Comment:
This effectively reimplements the map encoding, but it's not much code. If there is a better way to handle this, let me know!